frontend/src/App.js

```diff
@@ -1,25 +1,13 @@
 import React from 'react';
 import logo from './logo.svg';
```

Comment:
Since this is no longer being used, we can remove this line now

Reply:
Ok done.